<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogSingleTermAttr</class>
 <widget class="QDialog" name="DialogSingleTermAttr">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>424</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>端子属性</string>
  </property>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>201</y>
     <width>411</width>
     <height>161</height>
    </rect>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectRows</enum>
   </property>
   <attribute name="verticalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <column>
    <property name="text">
     <string>属性名</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>数值</string>
    </property>
   </column>
  </widget>
  <widget class="QCheckBox" name="CbShowTag">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>373</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>显示端子排代号</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BtnOK_2">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>370</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>端子排属性</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BtnOK">
   <property name="geometry">
    <rect>
     <x>201</x>
     <y>370</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BtnCancel">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>370</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>4</y>
     <width>181</width>
     <height>191</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="LbTerminalJpg">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>3</y>
      <width>130</width>
      <height>130</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>130</width>
      <height>130</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>130</width>
      <height>130</height>
     </size>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>:/new/prefix1/image/Term1_2_0_0.png</pixmap>
    </property>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>134</y>
      <width>161</width>
      <height>30</height>
     </rect>
    </property>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>91</x>
       <y>5</y>
       <width>21</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>右：</string>
     </property>
    </widget>
    <widget class="QComboBox" name="CbRightTerm">
     <property name="geometry">
      <rect>
       <x>115</x>
       <y>5</y>
       <width>41</width>
       <height>20</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>20</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>200</width>
       <height>16777215</height>
      </size>
     </property>
    </widget>
    <widget class="QComboBox" name="CbLeftTerm">
     <property name="geometry">
      <rect>
       <x>24</x>
       <y>5</y>
       <width>41</width>
       <height>20</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>20</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>200</width>
       <height>16777215</height>
      </size>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>1</x>
       <y>5</y>
       <width>21</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>左：</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="widget_UpDownTerm" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>162</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>91</x>
       <y>5</y>
       <width>21</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>下：</string>
     </property>
    </widget>
    <widget class="QComboBox" name="CbDownTerm">
     <property name="geometry">
      <rect>
       <x>115</x>
       <y>5</y>
       <width>41</width>
       <height>20</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>20</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>200</width>
       <height>16777215</height>
      </size>
     </property>
    </widget>
    <widget class="QComboBox" name="CbUpTerm">
     <property name="geometry">
      <rect>
       <x>24</x>
       <y>5</y>
       <width>41</width>
       <height>20</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>20</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>200</width>
       <height>16777215</height>
      </size>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>1</x>
       <y>5</y>
       <width>21</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>上：</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>4</y>
     <width>221</width>
     <height>191</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="3" column="1">
     <widget class="QLabel" name="LbTerminalFunction">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>20</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string> 端子排代号：</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="EdTerminalTag">
      <property name="enabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string> 项目代号：</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="EdDesignation">
      <property name="text">
       <string>1</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string> 端子序号：</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLabel" name="LbPartCode">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>20</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLabel" name="LbProTag">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>20</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string> 功能定义：</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string> 物料编码：</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
